public/style.css

```diff
@@ -225,6 +240,37 @@ li {
   width: 70vw;
 }
 
+.question-image-container {
+  max-width: 50%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 3rem;
+  
+}
+
+.question-image {
+  max-width: 260px;
+  border-radius: 10px;
+  border: 4px solid var(--dark-brown);
+}
+
+.question {
+  font-family: 'Press Start 2P', cursive;
+  font-size: 1.5rem;
+  letter-spacing: .3rem;
+  line-height: 2.2rem;
+  animation: animate 1s ease-in-out forwards;
```

Comment:
This is not looking so good style wise. I would suggest removing it, or altering it. Let me know if you want to brain storm transition animations!